app/helpers/form_helper.rb

```diff
@@ -354,7 +354,7 @@ def link_to_add_fields(name, f, association, partial = nil, options = {})
         { :f => builder }.merge(locals_option))
     end
     options[:class] = link_to_add_fields_classes(options)
-    link_to_function(name, "add_fields('#{options[:target]}', '#{association}', '#{escape_javascript(fields)}')".html_safe, options)
+    link_to_function(name, "add_fields('#{options[:target]}', '#{association}', '#{escape_javascript(fields)}', #{options[:direction] || 'append'})".html_safe, options)
```

Comment:
Sorry, I've miscalculated the `'`
```suggestion
    link_to_function(name, "add_fields('#{options[:target]}', '#{association}', '#{escape_javascript(fields)}', '#{options[:direction] || 'append'}')".html_safe, options)
```